internal/install/recipe_installer.go

```diff
@@ -86,6 +87,7 @@ func NewRecipeInstaller(ic types.InstallerContext, nrClient *newrelic.NewRelic)
 	rf := recipes.NewRecipeFilterRunner(ic, statusRollup)
 	spf := packs.NewServicePacksFetcher(&nrClient.NerdGraph, statusRollup)
 	cpi := packs.NewServicePacksInstaller(nrClient, statusRollup)
+	av := validation.NewAgentValidator(func(ctx context.Context, url string) ([]byte, error) { return utils.NewHTTPClient().Get(ctx, url) })
```

Comment:
This constructor invocation seems a bit complicated and is not easy to read.  Anything we can do to make it a little simpler?  Default function value?